### DISCUSSION TITLE
Vvveb on MAMP Server

### DISCUSSION THREAD

**Discussion by Aridio** [givanz/Vvveb#144] [Q&A]
I've installed vvveb locally using MAMP, but I've had a lot of URL errors, mainly because I've tried to install it on a sub directory, I'm not interested in installing it on the root directory (htdocs), is there any guide to achieve this? in the doc I didn't find anything.

Log error attached, thanks in advance!
[php_error.log](https://github.com/user-attachments/files/16047593/php_error.log)

**Comment by givanz** [comment]
Thanks for pointing out the missing info, I included it in the documentation https://docs.vvveb.com/installation#subdirectory-install

For subdirectory install you need to edit [env.php](https://github.com/givanz/Vvveb/blob/master/env.php#L47) and set `V_SUBDIR_INSTALL` as the subdirectory name, for example if installed in `/vvveb` folder you need to set it to

```php
defined('V_SUBDIR_INSTALL') || define('V_SUBDIR_INSTALL', '/vvveb');
```

If you are using apache you need to update [.htaccess](https://github.com/givanz/Vvveb/blob/master/.htaccess#L42) line 42 and [/public/.htaccess](https://github.com/givanz/Vvveb/blob/master/public/.htaccess#L43) line 43 and add the `subdirectory/` path before `index.php`

```
RewriteRule ^ vvveb/index.php [L]
```

**Reply by Aridio** [reply]
Thanks, now it works fine!
